test/server/cards/01_SOR/LeiaOrganaAllianceGeneral.spec.js

```diff
@@ -0,0 +1,129 @@
+                expect(this.player1).toBeAbleToSelectExactly([this.battlefieldMarine, this.fleetLieutenant, this.allianceXwing]);
+                expect(this.player1).not.toHaveEnabledPromptButton('Pass ability');
+
+                this.player1.clickCard(this.battlefieldMarine);
+                this.player1.clickCard(this.regionalGovernor);
+                expect(this.battlefieldMarine.exhausted).toBe(true);
+                expect(this.regionalGovernor.damage).toBe(3);
+                expect(this.battlefieldMarine.damage).toBe(1);
+
+                // second attack
+                expect(this.player1).toBeActivePlayer();
+                expect(this.player1).toBeAbleToSelectExactly([this.fleetLieutenant, this.allianceXwing]);
+                expect(this.player1).toHaveEnabledPromptButton('Pass ability');
+                this.player1.clickCard(this.allianceXwing);
+                expect(this.player1).toHaveEnabledPromptButton('Pass ability');
+                this.player1.clickCard(this.p2Base);
+                expect(this.allianceXwing.exhausted).toBe(true);
+                expect(this.p2Base.damage).toBe(2);
+
+                expect(this.player2).toBeActivePlayer();
+                expect(this.leiaOrgana.exhausted).toBe(true);
+            });
+
+            it('should allow passing the second attack', function () {
+                this.player1.clickCard(this.leiaOrgana);
+                this.player1.clickPrompt('Attack with a Rebel unit');
+                expect(this.player1).toBeAbleToSelectExactly([this.battlefieldMarine, this.fleetLieutenant, this.allianceXwing]);
+                expect(this.player1).not.toHaveEnabledPromptButton('Pass ability');
+
+                this.player1.clickCard(this.battlefieldMarine);
+                this.player1.clickCard(this.regionalGovernor);
+                expect(this.battlefieldMarine.exhausted).toBe(true);
+                expect(this.regionalGovernor.damage).toBe(3);
+                expect(this.battlefieldMarine.damage).toBe(1);
+
+                // second attack
+                expect(this.player1).toBeActivePlayer();
+                expect(this.player1).toBeAbleToSelectExactly([this.fleetLieutenant, this.allianceXwing]);
+                expect(this.player1).toHaveEnabledPromptButton('Pass ability');
+                this.player1.clickPrompt('Pass ability');
+
+                expect(this.player2).toBeActivePlayer();
+                expect(this.leiaOrgana.exhausted).toBe(true);
+            });
+        });
+
+        describe('Leia\'s deployed ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['atst', 'fleet-lieutenant', { card: 'battlefield-marine', exhausted: true }],
+                        spaceArena: ['alliance-xwing'],
+                        leader: { card: 'leia-organa#alliance-general', deployed: true }
```

Comment:
We should add something to the testing readme to show this is how to have a deployed leader. My first instinct would have been to include Leia in the ground arena

Reply:
I agree - are you still thinking of doing that readme?

Reply:
I wanted to but I'm not really sure I will be able to get to it soon

Reply:
Kk, we can figure out if we want to deal with it later or I can carve out some time. Seems like people are onboarding to tests without too much issue so it may not be urgent.